monkey/infection_monkey/network/smb_fingerprinter.py

```diff
@@ -127,25 +135,34 @@ def calculate(self):
         self.fields["bcc1"] = struct.pack("<i", len(self.fields["Data"]))[:2]
 
 
-class SMBFinger(HostFinger):
-    _SCANNED_SERVICE = "SMB"
+class SMBFingerprinter(IFingerprinter):
+    def get_host_fingerprint(
+        self,
+        host: str,
+        _ping_scan_data: PingScanData,
+        port_scan_data: Dict[int, PortScanData],
+        _options: Dict,
+    ) -> FingerprintData:
+        services = {}
+        smb_service = {
+            "display_name": SMB_DISPLAY_NAME,
```

Comment:
In HTTP and Elastic, we're hardcoding the string. In SSH, we have a `_SCANNED_SERVICE` class property that we're using. Here and in MSSQL, we have global constants that we're using. Let's decide on one for consistency.

Reply:
Agreed. This is a simple fix. Once all the fingerprinters are merged, we can make them consistent.